js/widgets/meterwidget.js

```diff
@@ -270,7 +274,7 @@ function MeterWidget() {
         return cell;
     };
 
-    this._save = () => {
+    _save = () => {
```

Comment:
Please use save() { .... } syntax to maintain consistency in code style across the project. This is not the official review. It is just the suggestions I got for my changes

Reply:
Hi, thanks for reviewing. But I have a question that if I use save instead of _save won't that mean I would have to change all instances of the class to use setup instead of _setup ? So I am wondering if that could introduce unnecessary errors.

Reply:
I meant it to be like  _save(..){...} rather than using arrow functions all over the file.

Reply:
I think I misunderstood you. I have made the changes. Thanks !